xdem/spatialstats.py

```diff
@@ -49,37 +48,18 @@
 from skimage.draw import disk
 
 from xdem._typing import NDArrayf
-from xdem.misc import deprecate
 
 with warnings.catch_warnings():
     warnings.filterwarnings("ignore", category=DeprecationWarning)
     import skgstat as skg
 
 
-@deprecate(
-    removal_version=Version("0.4"), details="xdem.spatialstats.nmad is being deprecated in favor of geoutils.stats.nmad"
-)
-def nmad(data: NDArrayf, nfact: float = 1.4826) -> np.floating[Any]:
```

Comment:
This function + decorator need to stay until version 0.4 (see message of the deprecation). Otherwise users will have an unexplained breaking change.